tests/helpers/test_get_id_by_name.py

```diff
@@ -66,8 +66,9 @@ def test_no_id(mocker):
 
 
 def test_multiple_resources_with_same_name(mocker):
-    """Verify get_id_by_name returns ID value of a found resource  when OpenStack query
-    returns multiple resources."""
+    """Verify get_id_by_name returns ID value of a found resource when OpenStack
+     query returns multiple resources.
+     """
```

Comment:
nitpick: the indentation of this doc-string is inconsistent with the others in this PR. I presume we are standardizing on having all lines in the doc-string align with the first character in the method name.